workloads/logging/README.md

```diff
@@ -27,15 +33,19 @@ Default: 3
 Number of Elasticsearch nodes.
 
 ### ES_STORAGE_CLASS
-Default: 'gp2'
+Default: 'gp3-csi'
 Storage class to use for the persistent storage. The faster the storage, better the Elasticsearch performance.
 
 ### ES_STORAGE_SIZE
 Default: `100G`
 Each data node in the cluster is bound to a Persistent Volume Claim that requests the size specified using this variable from the cloud storage.
 
+### ES_MEMORY_LIMITS
+Default: `16Gi`
```

Comment:
It looks like the default request and limit are the same. Do we really even need to set a limit here?

Reply:
We can keep this variable as long as defaults make sense, which is the case :)

Reply:
Agree, request=limit makes this a guaranteed pod which has implications in the context of preemption - guaranteed pods are preempted last, so there's probably several reasons why we have requests=limit for ES.

Reply:
I don't disagree that the values may be reasonable, etc. My thinking here was if we are testing the logging stack we shouldn't limit the backend data store as it is just something that we may hit (OOM of ES) which would negatively impact our tests but not be prudent to the results. I.e. if we cause ES to OOM it does NOT indicate a failed test. It just means we need to change those values and re-run. IMHO removing the limit just potentially saves us steps/debugging down the line.